kube_dns/tests/test_kube_dns.py

```diff
@@ -30,7 +32,7 @@ def __init__(self, content, content_type):
         self.headers = {'Content-Type': content_type}
 
     def iter_lines(self, **_):
-        for elt in self.content.split("\n"):
+        for elt in ensure_unicode(self.content).split("\n"):
```

Comment:
I am not sure if this is the right thing to do

Reply:
It's probably worth it to use the fixture format other prom checks use, for example https://github.com/DataDog/integrations-core/blob/a56a597c15a86e7e0307304afae15127ce7f897d/crio/tests/test_crio.py#L18-L31

WDYT?

Reply:
Yea probably

Reply:
@xvello what do you think now?